### ISSUE TITLE
Field class init can't set alias 

### ISSUE THREAD

**Issue by lgelmi** [kayak/pypika#441]
The alias argument won't actually set the alias attribute in a Field class, at least not through init.
Manually setting the field works fine.

Here's an example: 
```
from pypika import Field
bar = Field(name="foo", alias="bar")
print(bar.alias is None) 
>>> True
bar.alias = "foo"
print(bar.get_sql(with_alias=True))
>>> 'foo bar'
```
This happens with version 0.37.7.

Looking at the code I'd say this is due to the Field diamond inheritance: 
calling super() in `__init__` resolve to the JSON `__init__` whose first argument is value. 

I'd make a pull request changing the `super().__init__` call to pass both name and alias, but I'm not quite sure about the JSON part purpose...

**Comment by Apexal** [comment]
This seems fixed? Just tried with version 0.47.2:

```python
>>> from pypika import Field
>>> bar = Field(name="foo", alias="bar")
>>> print(bar.alias is None) 
False
>>> bar.get_sql(with_alias=True)
'foo bar'
```

@twheys

**Comment by lgelmi** [comment]
Seems like it, thanks! 

I can't seem to find any release note on the new version. Are they available somewhere?
